shapely/geometry/polygon.py

```diff
@@ -24,35 +24,39 @@ def _unpickle_linearring(wkb):
 
 class LinearRing(LineString):
     """
-    A closed one-dimensional feature comprising one or more line segments
+    A closed one-dimensional feature comprising one or more line segments.
 
     A LinearRing that crosses itself or touches itself at a single point is
     invalid and operations on it may fail.
+
+    Parameters
+    ----------
+    coordinates : sequence
+        A sequence of (x, y [,z]) numeric coordinate pairs or triples.
+        Also can be a sequence of Point objects.
```

Comment:
Consistency check: I'm assuming this can also come from something that provides the numpy array interface with appropriate shape.

Reply:
Indeed, the same as for a LineString. I made their docstring entries for this parameter match.